crates/parry3d/benches/common/macros.rs

```diff
@@ -72,7 +72,7 @@ macro_rules! bench_method_gen (
 
                 unsafe {
                     let val: $tres = test::black_box($arg.get_unchecked(i).$method($(unref($args.get_unchecked(i)),)*));
-                    drop(val);
+                    let _ = val;
```

Comment:
Why was that change needed?

Reply:
using `drop` results in a warning:

```rs
warning: calls to `std::mem::drop` with a value that implements `Copy` does nothing
   --> crates/parry3d/benches/bounding_volume/../common/macros.rs:75:21
```

`Aabb` and `BoundingSphere` are concerned. I don´t think this is an opportunity to make benchmarks more accurate (if we're unexpectedly copying).

Actually I'm not exactly sure why this drop is necessary. As we're in an unsafe call, I guess it might be useful, but not really used in current usage. It doesn´t hurt being here I guess.